### PR TITLE
Conflicts ReadFeed using correct pipeline

### LINKED ISSUE #204: Conflicts Feed fails with missing Partition Range
Conflict Read Feed operations fail with a missing Partition Key check. Conflict Read Feed should be treated the same as Document Read Feed operations in the request pipeline.

### PR DESCRIPTION
**Pull request by ealsur**
ReadFeed operation on Conflicts are failing after the latest Direct port. The Partition Key check is failing because it's not going through the PartitionKeyRangeHandler.

This fixes #204

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/Handler/CosmosRequestMessage.cs [hunk 1]
```diff
@@ -89,7 +89,7 @@ public virtual Stream Content
 
         internal bool IsPropertiesInitialized => this.properties.IsValueCreated;
 
-        internal bool IsDocumentFeedOperation => this.OperationType == OperationType.ReadFeed && this.ResourceType == ResourceType.Document && string.IsNullOrEmpty(this.PartitionKeyRangeId);
+        internal bool IsFeedOperation => this.OperationType == OperationType.ReadFeed && (this.ResourceType == ResourceType.Document || this.ResourceType == ResourceType.Conflict) && string.IsNullOrEmpty(this.PartitionKeyRangeId);
```
Comment (j82w) [inline Microsoft.Azure.Cosmos/src/Handler/CosmosRequestMessage.cs]: If this is feed operation why exclude Database and Container resource types?

Comment (j82w) [reply Microsoft.Azure.Cosmos/src/Handler/CosmosRequestMessage.cs]: Would a better name be IsPartitionedFeedOperation

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/src/Handler/CosmosRequestMessage.cs]: Database and container feed are expected to be partiton-proof (legacy). Needs to rework.
Till then lets use current one How about renaming appropriately for now.

Comment (ealsur) [reply Microsoft.Azure.Cosmos/src/Handler/CosmosRequestMessage.cs]: Renamed

**Pull request merged**